TestCases/strayalien/0079-feel-string-function/0079-feel-string-function-test-01.xml

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Contributed to DMN TCK by StrayAlien -->
+<testCases xmlns="http://www.omg.org/spec/DMN/20160719/testcase" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation=""  xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <modelName>0079-feel-string-function.dmn</modelName>
```

Comment:
I believe this "0079-feel-string-function" test case will be very difficult to agree on.

- I don't recall if date-related normalization are imposed by the spec
- being FEEL space insensitive, I doubt how we could agree on a single/unique output of a FEEL expression like `string([1, 2, 3, "foo"])`; on pragmatic terms this test case would impose it to be `[1, 2, 3, "foo"]` only, but `[ 1 , 2 , 3 , "foo" ]` would actually be a valid output too

if my understanding is correct, my personal preference would be to steer away for imposing a single/unique value check of the string format representation.

Reply:
I'm only trying to say that the test could consider a different strategy, to avoid checking for a single/unique output, when the specification allows for _several_ possible.
The test should steer a way to impose 1 and only value, when the specification allows for several. 

For example, the test could be rewritten with a more "complex" expression, still testing the `string()` function which is what you want, but something like `replace(string([1, 2, 3, "foo"]), "[\s\u00A0]+", "")` or ~~similar, so to avoid imposing the number of spaces, and checking just the part of the string which we know should indeed be part of the output regardless of the variants allowed by the spec! ;)